> b ThrowException
> r
> bt
* thread #1, [..] stop reason = breakpoint 1.1
  * frame #0: [..] program.kexe`ThrowException
    frame #1: [..] program.kexe`kfun:main$lambda-0#internal at kt42208-2.kt:3:18
    frame #2: [..] program.kexe`kfun:$main$lambda-0$FUNCTION_REFERENCE$0.invoke#internal(_this=[..]) at kt42208-1.kt:2:5
    frame #3: [..] program.kexe`kfun:$main$lambda-0$FUNCTION_REFERENCE$0.$<bridge-UNN>invoke(_this=[..]){}kotlin.Nothing#internal at kt42208-1.kt:2:5
    frame #4: [..] program.kexe`kfun:#main(){} at kt42208-1.kt:2:5
    frame #5: [..] program.kexe`Konan_start(args=[..]) at kt42208-1.kt:1:1
    frame #6: [..]
    frame #7: [..]
> q

